app/controllers/request-to-go-live/organisation-address/post.controller.js

```diff
@@ -26,27 +26,27 @@ const clientFormIds = {
 
 const validationRules = [
   {
-    field: clientFormIds.addressLine1,
+    field: clientFieldNames.addressLine1,
     validator: validateMandatoryField,
     maxLength: 200
```

Comment:
These seem to be out of sync with merchant_details/post-edit-controller.js

Reply:
Good point, I've updated them